composable.yml

```diff
@@ -113,3 +113,14 @@
         openstack overcloud roles generate -o ~/roles_data.yaml --roles-path ~/roles {{ roles }}
       args:
         chdir: "/home/stack"
+
+    - block:
+      - name: create roles directory as required for infrared
```

Comment:
@kemmy22 instead of creating separate check for OSP 17, can you please update the roles-path to ~/virt/roles to all the possible occurrences in jetpack code that  will avoid additional checks

Reply:
@asyedham I think we can track that through a separate issue/change. What do you think? Since this change is for ceph & manila.

Reply:
ack